rosdep/python.yaml

```diff
@@ -9321,6 +9321,56 @@ python3-sparkfun-ublox-gps-pip:
   ubuntu:
     pip:
       packages: [sparkfun-ublox-gps]
+python3-spconv-pip:
+  debian:
+    pip:
+      packages: [spconv]
+  fedora:
+    pip:
+      packages: [spconv]
+  ubuntu:
+    pip:
+      packages: [spconv]
```

Comment:
@JWhitleyWork CI is catching that this key should come after the others due to lexicographical sorting.